### PR TITLE
Implement DropShadow effect.

### PR DESCRIPTION
**Pull request by simeoncran**
This is currently turned off, however we do everything except adding the shadow to the LayerVisual so that the code is being exercised.
I have not output any issues yet for the cases we don't handle because we don't know how long before LayerVisual is fixed, and the set of issues might need changing when LayerVisual is finally working.

### CODE REVIEW

File: source/LottieReader/Serialization/Animatables.cs [hunk 1]
```diff
@@ -36,6 +36,7 @@ sealed partial class LottieCompositionReader
             CreateAnimatableParser((in LottieJsonElement element) => element.AsDouble() ?? 0);
 
         static readonly AnimatableParser<Opacity> s_animatableOpacityParser = CreateAnimatableParser(ParseOpacity);
+        static readonly AnimatableParser<Opacity> s_animatableOpacityParserByte = CreateAnimatableParser(ParseOpacityByte);
```
Comment (simeoncran) [inline source/LottieReader/Serialization/Animatables.cs:39]: Maybe this should called s_animatableOpacityByteParser

File: source/LottieToWinComp/CompositionObjectFactory.cs [hunk 2]
```diff
@@ -308,9 +308,6 @@ static float Clamp(float value, float min, float max)
 
         static Sn.Vector2 ClampedVector2(float x, float y) => Vector2(Clamp(x, 0, 1), Clamp(y, 0, 1));
```
Comment (derobinsMSFT) [inline source/LottieToWinComp/CompositionObjectFactory.cs:309]: >static Sn.Vector2 [](start = 8, length = 17)

A couple other methods in this class also have equivalents in `ConvertTo`.

**Review by derobinsMSFT — DISMISSED** [review]
:shipit:

**Review by derobinsMSFT — APPROVED** [review]
:shipit:

**Pull request merged**